app/views/newflow/login_signup/student_signup_form.html.erb

```diff
@@ -40,14 +40,16 @@
 
                         <div class="fb-share-button">
                             <%= link_to newflow_auth_path(:facebooknewflow),
-                                        class: 'facebook btn' do %>
+                                        class: 'facebook btn',
+                                        :data => {:ga_category => 'Account Creation', ga_action: 'Click', ga_label: '3B-Facebook'} do %>
```

Comment:
Don't we only want to track **successful** logins with facebook? Otherwise, this will only track how many people **tried** to log in with facebook. Same for google.

Reply:
How would I detect a failed login? It would not return to one of our pages would it? I could check if the user got to the profile page via facebook or google as the referrer.

Reply:
Also the requirements state capturing clicking the Facebook and Google buttons.

Reply:
Since we have another way of tracking successful signups (the Security Logs), then just tracking clicks regardless of the outcome is another thing we want to track I suppose, it makes sense, I just wanted to be sure that was the case. All good.